client/external_match_client/client.go

```diff
@@ -14,6 +15,32 @@ const (
 	apiKeyHeader   = "X-Renegade-Api-Key"
 )
 
+// ExternalMatchBundle is the application level analog to the ApiExternalMatchBundle
+type ExternalMatchBundle struct {
+	MatchResult  *api_types.ApiExternalMatchResult
+	SettlementTx *SettlementTransaction
+}
+
+// SettlementTransaction is the application level analog to the ApiSettlementTransaction
+type SettlementTransaction struct {
+	Type string
+	To   geth_common.Address
+	Data []byte
+}
+
+// toSettlementTransaction converts an ApiSettlementTransaction to a SettlementTransaction
+func toSettlementTransaction(tx *api_types.ApiSettlementTransaction) *SettlementTransaction {
+	// Parse a geth address and bytes data from hex strings
+	to := geth_common.HexToAddress(tx.To)
+	data := geth_common.FromHex(tx.Data)
+
+	return &SettlementTransaction{
+		Type: tx.Type,
+		To:   to,
+		Data: data,
+	}
+}
```

Comment:
super small nit: I would choose between `Tx` or `Transaction` and stick to one

Reply:
not sure what you mean?